cmd/run-linter/run-linter.go

```diff
@@ -135,7 +133,7 @@ func main() {
 
 		for _, r := range results {
 			if _, err = f.WriteString(r.String() + "\n"); err != nil {
-				grip.Error(err)
+				fmt.Fprintf(os.Stderr, "%v", err)
```

Comment:
Do we need to change this since we have to vendor grip anyways?

Reply:
I want run-linter to be the same between al repos to avoid drift in tooling.